l1-contracts/test/decoders/Decoder.t.sol

```diff
@@ -80,27 +50,21 @@ contract DecoderTest is DecoderBase {
 
     // Using the FULL decoder.
     (
-      uint256 l2BlockNumber,
-      bytes32 startStateHash,
-      bytes32 endStateHash,
-      bytes32 publicInputsHash,
+      bytes32 diffRoot,
```

Comment:
We might want to talk again around getting rid of the full decoder.

Reply:
Yes, as I mentioned above would do it in the https://github.com/AztecProtocol/aztec-packages/issues/3938 in which we'll update the body hashing.